src/app.module.ts

```diff
@@ -0,0 +1,45 @@
+import { ApolloDriver, ApolloDriverConfig } from '@nestjs/apollo';
+import { Module } from '@nestjs/common';
+import { GraphQLModule } from '@nestjs/graphql';
+import {
+	ApolloServerPluginLandingPageLocalDefault,
+	ApolloServerPluginLandingPageProductionDefault,
+} from 'apollo-server-core';
+import { GraphQLError, GraphQLFormattedError } from 'graphql';
+import { PrismaModule } from '../prisma/prisma.module';
+
+@Module({
+	imports: [
+		GraphQLModule.forRoot<ApolloDriverConfig>({
+			driver: ApolloDriver,
+			playground: false,
+			plugins: [
+				// Install a landing page plugin based on NODE_ENV
+				process.env.NODE_ENV === 'production'
+					? ApolloServerPluginLandingPageProductionDefault({
+							footer: false,
+					  })
+					: ApolloServerPluginLandingPageLocalDefault(),
+			],
+			typePaths: ['./**/*/*.graphql'],
+			//   typeDefs: (() => {
```

Comment:
Any reason these are in-line and not block?

Reply:
Alternatively, can this code be removed?

Reply:
Technically, yes. The primary motivation of leaving it in is to have it available as a reference, but I can see how that muddles the codebase. Let me think about it.

Reply:
Noted
Committing this is fine but definitely something to think about moving forward

Reply:
I've removed the comments as I have an external reference for this specifically